apps/site/lib/site_web/templates/fare/retail_sales_locations.html.eex

```diff
@@ -8,10 +8,31 @@
 
       <h2>Find a Store Near You</h2>
       <p></p>
-      <%= form_for @conn, @conn.request_path<>"#address-search-input", [as: :location, method: :get, class: "form"], fn _f -> %>
-        <%= input_location(%InputLocation{id: "address-search-input", name: :location, name_index: :address,
-                                          address: @address, address_error: get_flash(@conn, :info), submit: true,
-                                          required: true, placeholder: "Enter a location", button_class: ""}) %>
+      <div class="transit-near-me">
+        <%= form_for @conn, @conn.request_path, [as: :location, method: :get, class: "form m-transit-near-me__input"], fn f -> %>
+          <%=
+            SiteWeb.PartialView.render(
+              "_search_input.html",
+              form: f,
+              field: :address,
+              autocomplete?: true,
+              prefix: "transit-near-me",
+              aria_label: "",
+              placeholder: "Enter a location",
+              value: input_value(@search_position)
+            )
+          %>
+          <%= Phoenix.HTML.Form.hidden_input f, :latitude,
+                                            value: assigns[:latitude], id: "search-transit-near-me__latitude" %>
+          <%= Phoenix.HTML.Form.hidden_input f, :longitude,
```

Comment:
**praise:** Thanks for using a more or less conventional form here, it works great when navigating from a URL with lat/lon/address parameters :sparkles: